requirements.txt

```diff
@@ -1,10 +1,11 @@
-dateutil
+python-dateutil
+stripe
+foolscap
 Fabric
 filepath
 jinja2
 mock
 pyOpenSSL
 simplejson
 twisted
-
-https://tahoe-lafs.org/source/tahoe-lafs/deps/tahoe-dep-sdists/txAWS-0.2.1.post5.tar.gz
+https://tahoe-lafs.org/source/tahoe-lafs/deps/tahoe-lafs-dep-sdists/txAWS-0.2.1.post5.tar.gz
```

Comment:
Ah, also, I noticed some pip-invoking code that does `--find-links https://tahoe-lafs.org/deps/`.  That page includes the correct link to txAWS-0.2.1.post5.  I wonder if someone has the intention that the full URL of this archive be considered an implementation detail.  If that's the case, this line should really just be `txAWS` (or `txAWS==0.2.1.post5`?) and the correct `--find-links` argument should also be used.